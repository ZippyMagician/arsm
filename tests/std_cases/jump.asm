.loop
    jmp :helper
    chr eh
    jmp :loop
..helper
    mov eh in
    jz eh :end
    ret
..end
    hlt 0