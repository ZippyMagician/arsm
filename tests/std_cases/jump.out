this is the very long and complex input... good luck